contrib/platform/test/com/sun/jna/platform/win32/PdhTest.java

```diff
@@ -155,4 +159,54 @@ private static String makeCounterPath(Pdh pdh, PDH_COUNTER_PATH_ELEMENTS pathEle
         
         return Native.toString(szFullPathBuffer);
     }
+    
+    @Test
+    public void testLookupPerfIndex() {
+        // Index 238 is "Processor"
```

Comment:
The tests are locale specific (it failed on my initial run on a german locale). Please guard the critical parts with `AbstractWin32TestSupport.isEnglishLocale`. See here for an example:

https://github.com/java-native-access/jna/blob/028fff09fd473e2fe0fae8d781d2f754fe13daac/contrib/platform/test/com/sun/jna/platform/win32/Win32ExceptionTest.java#L33